misk/src/main/kotlin/misk/web/extractors/RequestBodyParameterExtractor.kt

```diff
@@ -17,7 +18,12 @@ internal class RequestBodyParameterExtractor(
   private val parameter: KParameter,
   private val unmarshallerFactories: List<Unmarshaller.Factory>
 ) : ParameterExtractor {
-  override fun extract(webAction: WebAction, request: Request, pathMatcher: Matcher): Any? {
+  override fun extract(
+    webAction: WebAction,
+    request: Request,
+    webSocket: WebSocket?,
```

Comment:
should a `WebSocket : WebAction`? `WebAction` appears to just be a marker interface

Reply:
I don't think that quite works - a ``WebAction`` is a target of dispatch, the ``WebSocket`` is a property of the request. But it does feel a little sad to expose web sockets to all parameter extractors, when the only one that ever will every use it is one that just returns the socket directly.  Might make more sense to make the ``WebSocket`` a property of the incoming request rather than a top-level parameter to ``extract``